workers/src/main/java/edu/cornell/testoutputstream/KafkaTestOutputStream.java

```diff
@@ -31,28 +30,25 @@ class KafkaTestOutputStream implements TestOutputStream {
 
     /**
      * Creates a new TestOutputStream
-     * @param kafkaAddress the address of the Kafka cluster
      */
-    KafkaTestOutputStream(@NonNull String kafkaAddress) {
+    KafkaTestOutputStream() {
         //Assign topicName to hostname
-        if (Main.DEBUG_MODE) {
-            topicName = "localhost";
-        } else {
-            try (BufferedReader inputStream = new BufferedReader(
-                    new InputStreamReader(
-                            Runtime.getRuntime().exec("hostname").getInputStream()))) {
-                topicName = inputStream.readLine();
-            } catch (IOException e) {
-                topicName = "error";
-                LOGGER.error("Error getting hostname from container", e);
-                System.exit(1);
-            }
+        try (BufferedReader inputStream = new BufferedReader(
+                new InputStreamReader(
+                        Runtime.getRuntime().exec("hostname").getInputStream()))) {
+            topicName = inputStream.readLine();
+        } catch (IOException e) {
+            topicName = "error";
+            LOGGER.error("Error getting hostname from container", e);
+            System.exit(1);
         }
 
+        LOGGER.info("topicName = " + topicName + " kafkaAddress = " + "kafka:29092");
+
         // create instance for properties to access producer configs
         Properties properties = new Properties();
         //Assign localhost id
-        properties.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaAddress);
+        properties.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka:29092");
         //Set acknowledgements for producer requests.
```

Comment:
currently hard coded kafkaAddress values

Reply:
This should be added as a private constant to the top of the class (but it's fine for it to be constant for now)

Reply:
I will make this change.